decidim-collaborative_texts/spec/forms/decidim/collaborative_texts/admin/document_form_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module CollaborativeTexts
+    module Admin
+      describe DocumentForm do
+        subject(:form) { described_class.from_params(attributes).with_context(context) }
+
+        let(:organization) { create(:organization) }
+        let(:context) do
+          {
+            current_organization: organization,
+            current_component:,
+            current_participatory_space: participatory_process
+          }
+        end
+        let(:participatory_process) { create(:participatory_process, organization:) }
+        let(:current_component) { create(:component, participatory_space: participatory_process) }
+        let(:title) do
+          Decidim::Faker::Localized.sentence(word_count: 3)
```

Comment:
This is the only place i have seen you actually use i18n for title. ( nothing to do here, just a comment )

Reply:
Actually,  there's something to do. It should not be localized